### ISSUE TITLE
Create an ability to remove inactive devices

### ISSUE THREAD

**Issue by shepilov** [linagora/twake-workplace#232]

**Comment by hoangdat** [comment]
We had it for user in `Settings`
<img width="1782" alt="image" src="https://github.com/linagora/twake-workplace/assets/6462404/6eca566b-9b16-4d97-8192-3db3238ba356">

**Comment by guimard** [comment]
> We had it for user in `Settings` 

Yes but here we have to avoid what happened to our platform: inactive devices creates "device_inbox" messages which make grows a lot Matrix DB.

Algorithm idea :
 - never clean the last 2 devices per user _(last TC web device, last TC app device and, if any, last non-TC device ; cleaned only on user purge, else this user will loose all messages)_
 - clean all inactive devices > 1 year
 - clean all inactive Twake devices > 3 months (because Twake has a recovery system)
 - clean all inactive Twake-Chat web devices > few weeks
 - clean inactive user => to be defined

**Comment by hoangdat** [comment]
> clean inactive user 

We should announce user about it (reminder email, messsage)

**Comment by guimard** [comment]
> > clean inactive user
> 
> We should announce user about it (reminder email, messsage)

Sure, but it's a separated issue, not for Chat only => @shepilov

**Comment by hoangdat** [comment]
Current logic for Device name:
```
'Twake Chat ${isWeb ? 'web' : Platform.operatingSystem}${kReleaseMode ? '' : 'Debug'}';
```

**Comment by guimard** [comment]
> Current logic for Device name:
> 
> ```
> 'Twake Chat ${isWeb ? 'web' : Platform.operatingSystem}${kReleaseMode ? '' : 'Debug'}';
> ```

OK, thanks. If you change something here, remember to open an issue for me also

**Comment by guimard** [comment]
Validated
